contrib/check-for-api-changes.sh

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env bash
+#
+# Checks the public API of crates, exits with non-zero if there are currently
+# changes to the public API not already committed to in the various api.txt
+# files.
+
+set -e
+
+# Crates in this repo to check.
+CRATES="bitcoin hashes"
```

Comment:
In a3dde3e716413a0606540fbf9c452a6bc63c4086:

`CRATES` is no longer used.

Reply:
ouch! `shellcheck` where were you?